lib/susedistribution.pm

```diff
@@ -153,6 +153,9 @@ sub init_cmd {
     if (check_var('INSTLANG', "fr_FR")) {
         $testapi::cmd{next} = "alt-s";
     }
+    if (check_var('FLAVOR', 'Online')) {
```

Comment:
This is added in the general susedistribution.pm file. In this change, if flavor is online, then will it always go to expert partitioner. Does this works for all 15sp2 online installations? I am afraid some tests do not go into this step. Do you need to add more constraint into this condition?

Reply:
I have not touched this file, but looks like it is just an enum for hotkey during installation, rather than running into expert partition. If it is the case, where is the change in your tests? why not submit together. 

Also, is the hotkey identical in full and online installation? if yes, we need not specify the online flavor only, as the command hotkey usually varies from language.

Reply:
the case(lib/partition_setup.pm) not need change,since it's only run "send_key $cmd{expertpartitioner}". 

expertpartitioner shotcut key is different for full installation and online installation,the old only ok for full installation.so I add this for online installation.